### PR TITLE
Commiting How to update SoftNAS KB.

### PR DESCRIPTION
**Pull request by maxim-clc**
Commiting How to update SoftNAS KB.

### CODE REVIEW

**Comment by rpdubose** [comment]
@maxim-clc FYI on the issues associated w/ this PR above.  Let us know if you have questions.

Thanks,

File: Servers/how-to-update-softnas.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+{{{
+  "title": "SoftNAS: How to update",
+  "date": "11-11-2015",
+  "author": "Maxim Volkov",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": true
```
Comment (chrislittle) [inline Servers/how-to-update-softnas.md]: As simply a way to update a 3rd party product I don't think this should be a sticky/must read kb.

File: Servers/how-to-update-softnas.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
+{{{
+  "title": "SoftNAS: How to update",
+  "date": "11-11-2015",
+  "author": "Maxim Volkov",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
```
Comment (chrislittle) [inline Servers/how-to-update-softnas.md]: Please code in html or markdown not a mix. Recommend markdown.  Update your entire kb

File: Servers/how-to-update-softnas.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+### Introduction
+
+<p>SoftNAS Cloud is a software behind virtual NAS appliance provided by SoftNAS, LLC - CenturyLink ecosystem partner. The software gets updated on a regular bases to receive bug fixes and new features. This brings the necessity to periodically upgrade your SoftNAS. Here we will describe this procedure.</p>
+
+### Prerequisites
+
+<div>
+<ol>
+<li>A SoftNAS appliance was [ordered](../ecosystem-partners/marketplace-guides/getting-started-with-softnas-cloud-file-gateway-partner-template.md) and [set up](../using-a-software-defined-virtual-nassan-on-centurylink-cloud.md)  in CenturyLin Cloud.</li>
```
Comment (chrislittle) [inline Servers/how-to-update-softnas.md]: Bad links see the readme

**Comment by mvolkov** [comment]
Fixed markdown and link errors

**Comment by mvolkov** [comment]
Fixed markdown and link errors as per earlier request.

**Comment by mvolkov** [comment]
Corrections been made and article is ready.

**Comment by jim-phillips** [comment]
@maxim-clc or @mvolkov Please remove the node_modules directory

**Comment by mvolkov** [comment]
The "node_modules" directory was removed as requested.

**Comment by chrislittle** [comment]
i still see a package.json file you are trying to commit. remove that.

**Comment by mvolkov** [comment]
package.json removed

**Comment by jim-phillips** [comment]
@mvolkov I still see it.  Did you commit and sync?

**Comment by maxim-clc** [comment]
Fixes are implemented.

File: Servers/how-to-update-softnas.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
+node_modules{{{
```
Comment (jim-phillips) [inline Servers/how-to-update-softnas.md]: @mvolkov delete 'node_modules'

**Comment by maxim-clc** [comment]
Edited

**Comment by maxim-clc** [comment]
All checks passed.

**Pull request merged**